### PR TITLE
Added play state logic.

### PR DESCRIPTION
**Pull request by MitchelSmith**
Logic for #4

### CODE REVIEW

File: src/Pages/HomePage.js [hunk 1]
```diff
@@ -3,11 +3,24 @@ import AppBar from '../components/AppBar';
...
+
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      play: false
+    };
+  }
+
+  toggleAudio = () => { this.setState({ play: !this.state.play }); };
```
Comment (maryehuerta) [inline src/Pages/HomePage.js]: 👍  Love that you passed a function down through props + es6 syntax on this 💯

File: src/Pages/HomePage.js [hunk 2]
```diff
@@ -3,11 +3,24 @@ import AppBar from '../components/AppBar';
...
+
   render() {
     return (
       <div>
         <AppBar />
-        <PlaylistDrawer/>
+        <PlaylistDrawer
+        toggleAudio={this.toggleAudio.bind(this)}
```
Comment (kolten) [inline src/Pages/HomePage.js]: This is good, but we can make this even better: In ES6 JavaScript with fat arrow functions, you can pass in `this.toggleAudio`, as the arrow function already has `this` bind-ed to it. Good job though!

File: src/components/PlaylistDrawer.js [hunk 3]
```diff
@@ -77,7 +77,9 @@ function PermanentDrawerLeft(props) {
 						</List>
 					</Grid>
 					
-					<MusicPlayer/>
+					<MusicPlayer
+          toggleAudio={props.toggleAudio.bind(this)}
```
Comment (kolten) [inline src/components/PlaylistDrawer.js]: Same thing here, we're passing a prop down that's already been binded with it's own `this,` so we can pass down `props.toggleAudio`.

Comment (maryehuerta) [inline src/components/PlaylistDrawer.js]: Same things as kolten said about .bind()

**Comment by kolten** [comment]
💯

**Comment by doms** [comment]
great job! @MitchelSmith 🎉 

I can see why he used `bind` initially; almost every tutorial I've seen usually does something this in the constructor:

```js
constructor(props) {
  super(props)
  
  this.state = { ... }

  this.toggleAudio = this.toggleAudio.bind(this) // <------ I see this in most tuts
}
```
which would then allow for the function to be auto-binded like mentioned in the PR reviews. I noticed this constructor does not have this behavior 🤔 

@kolten @maryehuerta 

Is it not required? (Obviously, I am react noob, so it might not even be needed 😄 )

**Comment by kolten** [comment]
It's not wrong to use bind in the constructor, just more concise if you don't use it. This is one of the big features in ES6 compared to ES5.

**Pull request merged**